x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/http/retry/AlwaysRetryingResponseHandler.java

```diff
@@ -58,4 +58,8 @@ public InferenceServiceResults parseResult(Request request, HttpResult result) t
         }
     }
 
+    @Override
```

Comment:
Moved this class to the test directory since that's the only place it's referenced.